.github/workflows/python-tests.yml

```diff
@@ -21,117 +21,90 @@ jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
Since CI is currently taking a long time I thought it would be good to still run all builds even though 1 failed.
I can see the possibility where a PR breaks 2 tests from 2 different builds. I would prefer to know directly that I have 2 tests to fix than fixing 1 and then realise about the other one.

Reply:
Sure!